src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -525,27 +523,17 @@ export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
           <WithLazyMount shouldUse={Boolean(receiveAddress)} component={RecipientAddress} />
           <WithLazyMount shouldUse={!walletSupportsBuyAssetChain} component={ManualAddressEntry} />
           {maybeUnsafeTradeWarning}
```

Comment:
This should be removed, as well as the `maybeUnsafeTradeWarning` constant itself

Reply:
Indeed, gm ser

https://github.com/shapeshift/web/pull/6647/commits/916ecb47d6f7964089d596f97a9406fbe95c1d43